Google.PowerShell.IntegrationTests/Logging/Logging.GcLogMetric.Tests.ps1

```diff
@@ -61,3 +61,168 @@ Describe "Get-GcLogMetric" {
         { Get-GcLogMetric -Metric "non-existent-metric-name" -ErrorAction Stop } | Should Throw "does not exist"
     }
 }
+
+Describe "New-GcLogMetric" {
+    It "should work with -LogName" {
+        $r = Get-Random
+        $metricName = "gcps-new-gclogmetric-$r"
+        $logName = "gcps-new-gclogmetric-log-$r"
+        try {
+            New-GcLogMetric $metricName -LogName $logName
```

Comment:
Why are you not setting $createdMetric here? New-GcLogMetric does return the metric.